helm-chart/values.yaml

```diff
@@ -137,3 +137,20 @@ test:
   enabled: false
   schedule: '@daily'
   baseUrl: "http://127.0.0.1:7546"
+
+livenessProbe:
+  httpGet:
+    path: /health/liveness
+    port: jsonrpcrelay
+  initialDelaySeconds: 50
```

Comment:
The initialDelaySeconds seems high. Does your app need that time to initialize? Maybe closer to 20 for each?